packages/cli/src/ceramic-daemon.ts

```diff
@@ -456,13 +449,11 @@ export class CeramicDaemon {
       adminPinsRouter.postAsync('/:streamid', this.pinStream.bind(this))
       adminPinsRouter.deleteAsync('/:streamid', this.unpinStream.bind(this))
 
-      documentsRouter.postAsync('/', this.createDocFromGenesis.bind(this)) // Deprecated
       recordsRouter.postAsync('/', this.applyCommit.bind(this)) // Deprecated
```

Comment:
We should make sure to delete the `records` endpoints too while we're at this.  Not sure if there's a ticket for that or not.